tensorboard/plugins/pr_curve/summary.py

```diff
@@ -31,24 +31,19 @@ def op(
     labels,
     predictions,
     num_thresholds=200,
-    weights=None,
-    num_classes=1,
+    weight=1.0,
     display_name=None,
-    description=None):
-  """Computes multi-class PR summaries for a list of thresholds in `[0, 1]`.
+    description=''):
```

Comment:
Cool, I had not thought of that. The main reason I have used None but not objects as default function parameters is to avoid this utter silliness:

```
>>> def foo(my_list=[]):
...   my_list.append(':(')
...   return my_list
... 
>>> foo()
[':(']
>>> foo()
[':(', ':(']
>>> foo()
[':(', ':(', ':(']
```

but I had not thought about primitives vs None. Done.